.github/workflows/_linux-build.yml

```diff
@@ -136,10 +136,11 @@ jobs:
 
       - name: Calculate docker image
         id: calculate-docker-image
-        uses: pytorch/test-infra/.github/actions/calculate-docker-image@main
+        uses: pytorch/test-infra/.github/actions/calculate-docker-image@test-calculate-docker-image
```

Comment:
This is a temp change, right?

Reply:
Yes, just for testing purposes, not meant to be reviewed or merged 🙏